src/rich-text-editor.component.js

```diff
@@ -123,9 +124,9 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
 
   useEffect(() => {
     if (!hasSetInitialHTML && props.initialHTML) {
-      setHasSetInitialHTML(true)
       setHTML(props.initialHTML)
     }
+    setHasSetInitialHTML(true)
```

Comment:
even if there is no initialHTML I believe we want to set this?

Reply:
wait i think you totally misunderstood me. This code is not what I suggested. We should always call setHasSetInitialHTML(true). but we should call `setHTML(props.initialHTML, true)`